snakefmt/formatter.py

```diff
@@ -36,6 +37,10 @@
 )
 
 
+def comment_start(string: str):
+    return string[0] == "#" if string else False
```

Comment:
```suggestion
def comment_start(string: str) -> bool:
    return string.startswith("#")
```

Reply:
Would it make sense to rename to `is_comment`?

Reply:
Nice solution!

Reply:
I prefer keeping comment_start name because been using `is_` for functions testing token types from `tokenize`